salt/kubernetes-common/init.sls

```diff
@@ -0,0 +1,9 @@
+/etc/kubernetes/config:
+  file.managed:
+    - source:     salt://kubernetes-common/config.jinja
```

Comment:
`/etc/kubernetes/config` is installed by the `kubernetes-common` package, so we must explicitly install it here _before_ trying to write our own `config`. Otherwise the _rpm_ will overwrite it...